Concurrency/AI_interface.py

```diff
@@ -179,7 +179,7 @@ def representation_evauation(self):
             # # ray.get(storage)
             # ray.get(workers)
 
-    def position_ppo_testing(self):
+    def position_ppo_testing(self, remote_env=True):
```

Comment:
You can move the import ray at the top of the file into the individual methods in the AI_Interface. Either that or move this entire method to it's own file (which in my eyes is preferable since this is basically a model method and not a concurrency method).

Reply:
would you agree to move it to `Models/PositionModels/PPO_position_torch.py`? Do you want to keep this class architecture even if it has a single method and no attributes?